### PR TITLE
Elasticsearch2 update

### PR DESCRIPTION
**Pull request by tomsrocket**
More documentation and ES2

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,26 +1,30 @@
...
-    - logstash/conf:/opt/logstash/conf
-  links:
-    - elasticsearch
-
 elasticsearch:
-  image: elasticsearch:1.6
-  volumes:
-    - elasticsearch/config/templates:/usr/share/elasticsearch/config/templates
-    - elasticsearch/data:/usr/share/elasticsearch/data
+  image: elasticsearch:latest
```
Comment (webwurst) [inline docker-compose.yml:2]: Let's use explicit versioning like: `elasticsearch:2`

File: kibana/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,10 @@
+FROM kibana:latest
```
Comment (webwurst) [inline kibana/Dockerfile:1]: `kibana:4.2`

**Comment by tomsrocket** [comment]
Why? I got this from some official docker-elk image. I would rather use the newest and fix bugs if stuff breaks than stay on some specific old Version forever.

**Comment by webwurst** [comment]
Ok, go for it ;) But idea is just that you define a setup that _works_. You can always change to newer versions, test them and commit these if it works. If you use "latest" your setup may work one day but throws errors the next day. And your proud live demo goes down, others start filing issues and some might curse at you. If you use `elasticsearch:2.0` then you would also always use the newest of `2.0.x`.

**Pull request merged**